capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
```

Comment:
- [ ]  Hi, Here is the outcome I am getting when checking your codes.  Please can you check and see if you can fix it?

![Capture](https://user-images.githubusercontent.com/63972670/223664192-9fa6da85-24fb-496e-be09-6511dded4ba1.JPG)